microsoft/testsuites/kvm/kvm_unit_tests.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from pathlib import Path
+
```

Comment:
we can get rid of  new lines here

Reply:
`make fix` is automatically adding this.